Java-Control-Statements.md

```diff
@@ -0,0 +1,223 @@
+#Java Control Statements :
+
+In any programming language, the control over normal flow of program is necessary to make dynamic decisions at runtime and achieving desired results. `Control statements` manage the flow of execution of the program and decide the branching of logical flow. Java's program control statements can be put into following categories: `selection` , `iteration` and `jump statement` . `Selection statement` allows a program to choose different path of executions depending on an expression. `Iteration` allows a program to repeat one or more statements. `Jump statement` is used to transfer control to another part of the program. As `iteration` and `jump statements` are already covered in `Java-loops`, here we would discuss `Selection statements`.
```

Comment:
I think it will be appropriate to use backticks for codes and for emphasising general words like *Control statements*, *jump statements*, etc, _italics_ will do.